src/main/config/secor.common.properties

```diff
@@ -385,6 +385,12 @@ secor.kafka.upload_at_minute_mark.topic_filter=
 # What the minute mark is. This isn't triggered unless the topic name matches
 secor.upload.minute_mark=0
 
+# No of retries to upload the file. The retry attempts are exponential.
+secor.upload.retries=3
+
+# Backoff time to retry file upload
+secor.upload.retry.backoff=100
```

Comment:
How about name it 'secor.upload.retry.backoff.millis', so people know the unit of time.